### PR TITLE
Closes #94 - Fix bugs and add timer function

### LINKED ISSUE #94: Fix bugs and add timer function
Сreate an asynchronous stopwatch. Create Update method for timer names.  Fix buttons and edit page styles.

### PR DESCRIPTION
**Pull request by maryananovosad**

### CODE REVIEW

File: src/app/services/archive.service.ts [hunk 1]
```diff
@@ -20,42 +20,20 @@ export class ArchiveService implements OnInit {
...
-  createTask(task: TaskCreateJson) {
-    return this.http.post<TaskJson>(this.service.urlTask + 'CreateTask', task);
-  }
-
-  updateTask(task: Task) {
-    return this.http.put<TaskJson>(this.service.urlTask + 'CreateTask', task.convertToTaskJson());
-  }
 
+ 
+
```
Comment (YurasNazar) [inline src/app/services/archive.service.ts]: redundant new line

File: src/app/services/tasks.service.ts [hunk 2]
```diff
@@ -10,18 +13,46 @@ export class TasksService implements OnInit {
...
+    }
+
+    updateTask(task: TaskCreateJson) {
+        
+        return this.http.put<TaskCreateJson>(this.service.urlTask + 'UpdateTask', task);
+    }
+
+
+
+
```
Comment (YurasNazar) [inline src/app/services/tasks.service.ts]: redundant three new line!

File: src/app/app.module.ts [hunk 3]
```diff
@@ -138,3 +141,4 @@ export function jwtTokenGetter() {
 })
 export class AppModule { }
 
+
```
Comment (YurasNazar) [inline src/app/app.module.ts]: redundant new line!

File: src/app/services/archive.service.ts (hunk 1 above)
Comment (YurasNazar) [inline src/app/services/archive.service.ts]: redundant two new lines!

File: src/app/services/tasks.service.ts (hunk 2 above)
Comment (YurasNazar) [inline src/app/services/tasks.service.ts]: redundant new lines!

File: src/app/tasks/tasks.component.ts [hunk 4]
```diff
@@ -1,31 +1,39 @@
...
+import { timer, Subscription } from 'rxjs';
 
 @Component({
   selector: 'app-tasks',
   templateUrl: './tasks.component.html',
   styleUrls: ['./tasks.component.css'],
-  providers: [ArchiveService]
+  providers: [TasksService],
+
+
```
Comment (YurasNazar) [inline src/app/tasks/tasks.component.ts]: redundant new lines!

File: src/app/tasks/tasks.component.ts [hunk 5]
```diff
@@ -45,11 +53,24 @@ export class TasksComponent implements OnInit {
...
+      }
+    });
+  }
+
+  compare(a: String, b: String) {
+    return (a.toLowerCase() < b.toLowerCase() ? -1 : 1);
+  }
+
+
+
```
Comment (YurasNazar) [inline src/app/tasks/tasks.component.ts]: redundant new lines!

File: src/app/tasks/tasks.component.ts [hunk 6]
```diff
@@ -59,56 +80,104 @@ export class TasksComponent implements OnInit {
...
+    if (task.isRunning) {
+      task.currentSecond++;
+      task.hour = Math.floor(task.currentSecond / 3600);
+      task.minutes = Math.floor((task.currentSecond - (3600 * task.hour)) / 60);
+      task.seconds = Math.floor((task.currentSecond - (3600 * task.hour)) % 60);
+    }
+  }
+
+
+
```
Comment (YurasNazar) [inline src/app/tasks/tasks.component.ts]: redundant new lines!

File: src/app/tasks/tasks.component.ts [hunk 7]
```diff
@@ -45,11 +53,24 @@ export class TasksComponent implements OnInit {
...
   }
-
   ngOnInit() {
     this.getActiveTasks();
   }
 
+  filterByProperty(propertyName: string) {
+    this.tasks = this.tasks.sort((a, b) => {
+      switch (propertyName) {
+        case 'name': return this.compare(a.name, b.name);
```
Comment (roman-fay) [inline src/app/tasks/tasks.component.ts]: no point in having switch with only one case

Comment (maryananovosad) [reply src/app/tasks/tasks.component.ts]: Okey, I'll correct it

**Pull request merged**